src/TerminalFighter/MainMenu/MainMenu.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <chrono>
+#include <unistd.h>
+
+#include <SDL_ttf.h>
+#include <SDL2/SDL.h>
+
+#include "../TextArea/TextArea.h"
+
+namespace{
```

Comment:
I think this should public ENUM, then anyone interacting with the MainMenu can actually look at enum constants instead of ints.